test/test_all.py

```diff
@@ -153,6 +153,24 @@ def __init__(self, arch, container, address, *args, **kwargs):
                            "graph_test_15_01.dot"
                        ])
 
+testset += RegressionTest(["depgraph.py"], base_dir="analysis",
+                          products=[fname for fnames in (
+                              ["graph_test_%02d_00.dot" % test_nb,
+                               "graph_%02d.dot" % test_nb]
+                              for test_nb in xrange(1, 16))
+                                    for fname in fnames] +
+                          ["graph_test_04_01.dot",
+                           "graph_test_05_01.dot",
+                           "graph_test_06_01.dot",
+                           "graph_test_08_01.dot",
+                           "graph_test_09_01.dot",
+                           "graph_test_10_01.dot",
+                           "graph_test_13_01.dot",
+                           "graph_test_14_01.dot",
+                           "graph_test_14_02.dot",
+                           "graph_test_15_01.dot"
+                       ])
+
```

Comment:
I think there is a problem with generated tests:
There is an interference between tests generated by the `depgraph` and tests generated by the `influence` graph, as both have tests'name in common. Running regression tests on a multicpu machine will generate warnings:
```
Cleanning error: Unable to remove ..../test/analysis/graph_test_04_01.dot
Cleanning error: Unable to remove ..../test/analysis/graph_test_06_01.dot
Cleanning error: Unable to remove ..../test/analysis/graph_test_14_02.dot
```

Maybe you need to modify generated tests'names for example  `graph_dependence_test_13_01.dot` and `graph_influence_test_13_01.dot` to avoid such interference.